docker-compose.yml

```diff
@@ -0,0 +1,171 @@
+version: "3.7"
+
+networks:
+  ba-mojaloop-net:
+    name: ba-mojaloop-net
+
+services:
+  bulk-api-adapter:
+    build:
+      context: .
+      cache_from:
+        - mojaloop/bulk-api-adapter
+        - bulk-api-adapter
+    container_name: ba_bulk-api-adapter
+    command:
+      - "sh"
+      - "-c"
+      - "sh /opt/wait-for/wait-for-bulk-api-adapter.sh && node src/api/index.js"
+    ports:
+      - "3003:3003"
+    environment:
+      - LOG_LEVEL=info
+    volumes:
+       - ./docker/bulk-api-adapter/default.json:/opt/bulk-api-adapter/config/default.json
+       - ./docker/wait-for:/opt/wait-for
+    networks:
+      - ba-mojaloop-net
+    depends_on:
+      - central-ledger
+      - kafka
+      - objstore
+    healthcheck:
+      test: ["CMD", "sh", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:3003/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
+
+  objstore:
+    image: mongo:latest
+    container_name: ba_objstore
+    logging:
+      driver: none
+    ports:
+      - "27017:27017"
+    networks:
+      - ba-mojaloop-net
+    healthcheck:
+      test: mongo localhost:27017/test --quiet 1
+      interval: 10s
+      timeout: 10s
+      retries: 5
+      start_period: 40s
+
+  ml-api-adapter:
+    image: mojaloop/ml-api-adapter
+    container_name: ba_ml-api-adapter
+    command:
+      - "sh"
+      - "-c"
+      - "sh /opt/wait-for/wait-for-ml-api-adapter.sh && node src/api/index.js"
+    ports:
+      - "3000:3000"
+    environment:
+      - LOG_LEVEL=info
+    volumes:
+       - ./docker/ml-api-adapter/default.json:/opt/ml-api-adapter/config/default.json
+       - ./docker/wait-for:/opt/wait-for
+    networks:
+      - ba-mojaloop-net
+    depends_on:
+      - central-ledger
+      - kafka
+    healthcheck:
+      test: ["CMD", "sh", "-c" ,"apk --no-cache add curl", ";", "curl", "http://localhost:3000/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
```

Comment:
Does bulk-api-adapter require ml-api-adapter?

Reply:
Thanks @rmothilal. No it doesn't.